### DISCUSSION TITLE
** ERROR: H3D EXTERNAL LIBRARY NOT FOUND     ERROR TERMINATION

### DISCUSSION THREAD

**Discussion by anoopngmys** [OpenRadioss/OpenRadioss#1557] [Q&A]
Hi,

I followed the steps to download OpenRadioss to my Windows system and try running the Bird Strike on Windshield example problem. Encountered the error "Abort(2) on node 0 (rank 0 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 2) - process 0".

The engine.out shows "** ERROR: H3D EXTERNAL LIBRARY NOT FOUND     ERROR TERMINATION"

Please help me to resolve the issue.

Thanks.

**Comment by sebastienVilleneuve** [comment]
Hi @anoopngmys 

Did you set RAD_H3D_PATH env variable ?

cf
https://github.com/OpenRadioss/OpenRadioss/blob/main/INSTALL.md

Thanks,
Sébastien

**Reply by anoopngmys** [reply]
Hi @sebastienVilleneuve,

I have downloaded pre-compilled openradioss files in C drive (please look below snap for OPENRADIOSS_PATH).
![OPENRADIOSS_PATH_capture](https://github.com/OpenRadioss/OpenRadioss/assets/130025938/46b73e88-3cf9-4dd2-81e5-fd18dd298b22)


I have properly set RAD_H3D_PATH env variable path (please look below snaps for run_mpi.bat and RAD_H3D_PATH). 

![run_mpi_capture](https://github.com/OpenRadioss/OpenRadioss/assets/130025938/794316cc-247c-4377-8284-76594a80e3ca)

![RAD_H3D_PATH_Capture](https://github.com/OpenRadioss/OpenRadioss/assets/130025938/e4de561a-f919-49c7-aab3-9f277d70c9d6)

Also please find the snap of engine.out file for the error I am getting.

![engine_out_Capture](https://github.com/OpenRadioss/OpenRadioss/assets/130025938/9c940c92-c4ea-47f5-a6d5-694220a3f12f)

Kindly help me in resolving this issue.

Thanks,
Anoop
